src/js/bail-trends.js

```diff
@@ -3,12 +3,19 @@ import { BailRateMap, BailPostingMap, SwitchableMap } from "./classes/Map.js";
 import { ScatterPlot, DistributionGraph } from "./classes/Graph.js";
 import {
   BAIL_RATE_DATA,
+  PA_BAIL_CASES,
+  PA_ROR_CASES,
+  PA_TOTAL_CASES,
+  PA_AVG_POSTING_RATE,
   ROR_RATE_DATA,
   BAIL_POSTING_DATA,
   BAIL_CASES_SCATTER_PLOT,
   MDJ_DATA
 } from "./data.js";
 
+const PA_BAIL_RATE = Math.round(PA_BAIL_CASES / PA_TOTAL_CASES * 100 * 10) / 10;
```

Comment:
Why `* 10` and then `/ 10`? Rounding stuff?

Reply:
Yeah it rounds to the tenths decimal place. Is there a better way to do that (didn't want to use .toFixed since it casts to string, but I guess I can cast back to Number)

Reply:
Yeah probably using `toFixed` and casting back to a float is your best bet? I think that'll just be a bit easier to understand as well.